docs/using-radicle/fetching-and-merging.md

```diff
@@ -0,0 +1,149 @@
+contributions][tv] section.
+
+If you can see your a tracked peer's changes after adding them as a remote, this
+means they have been successfully replicated to your machine. If you have added
+a peer as a remote and can't see their changes, head over to our
+[Troubleshooting][tr] section.
+
+## Fetching changes 
+
+If you're interested in merging these changes into one of your branches, you'll
+need to fetch these commits into your working copy [TODO: section on working
+copy]. To make sure you can fetch or push changes, you need to add the
+[`git-rad-remote` helper][gs] to your `$PATH`.
+
+Navigate to your woking copy in a terminal. Here you can inspect the remotes of
+your project by running the `git remote` command. For example, a project that
+tracks multiple people might look something like this.
+
+```
+$ git remote show
+fintohaps@hybaapjdrhj8mg4mathpwjaeuktfzxd8uqfpjknp65fyncehpxxzze
+merle@hyb1exienq5txww75rgdftuf6j78qehtigma6hn19zt8gz5pu5k73g
+rad
+xla@hydxmk8cx8mcbijp6kgb8htm6jdhpstiow3kbezxp438adybaaqzgy
+```
+
+As you can see, the format of a remote handle is:
+
+```
+<display name>@<Device ID>
+```
+
+where the display name is the name you see on a user's profile page in Upstream.
+
+
+**Note**: You may be used to the concept of setting up an `origin` remote for a
+project when pushing to a centrally hosted platform like GitHub. This allows you
+to fetch and push changes from the server that hosts your project. In Radicle,
+these remotes point to the Git monorepo on your machine that stores all of your
+Radicle data. They allow you to fetch changes from your peers and push your
+changes. This means that fetching and pushing to remotes are _local first_
+actions. To read more about how Radicle is built on Git see our [How it
+Works][gi] section.
+
+To fetch the changes of a specific remote, run the `git fetch` command with
+their remote handle:
+
+```
+$ git fetch merle@hyb1exienq5txww75rgdftuf6j78qehtigma6hn19zt8gz5pu5k73g
+remote: Enumerating objects: 7, done.
+remote: Counting objects: 100% (7/7), done.
+remote: Compressing objects: 100% (3/3), done.
+remote: Total 6 (delta 0), reused 6 (delta 0), pack-reused 0
+Unpacking objects: 100% (6/6), 1.50 KiB | 1.50 MiB/s, done.
+From rad://hwd1yre8wtxufyqnt8xzxt8ps3n7ww8qzpfmud5pibarumshawnpcrjzjme
+ * [new ref]         hyb1exienq5txww75rgdftuf6j78qehtigma6hn19zt8gz5pu5k73g/heads/main -> merle/main
+```
+
+A successful fetch will have a similar output to the example above and show that
+the reference is now located as a remote branch (e.g. `merle/main`). We can
+confirm this is under a remote branch by entering the following command:
+
+```
+$ git branch -a
+main
+remotes/merle/main
+remotes/rad/main
+```
+
+<details>
+<summary>**Where did the
+`@@hyb1exienq5txww75rgdftuf6j78qehtigma6hn19zt8gz5pu5k73g` go?**</summary>
+The suffix of the remotes is disappearing and is a known
+[issue](https://github.com/radicle-dev/radicle-upstream/issues/1389).
```

Comment:
@MeBrei once your fix lands could you update this section please? :bow:

Reply:
:+1: Will add it to the issue!

Reply:
removed/updated in 12b1094

Reply:
It must have been fate that I had you in the copy and you were the one to fix it :smile: